### DISCUSSION TITLE
m_ssl_gnutls.so file does not exist

### DISCUSSION THREAD

**Discussion by James-Seely-Mirrors** [inspircd/inspircd#1845] [Support]
**Description**
inspircd was installed from source. everything functions as normal until ssl_gnutls module in enabled with ./configure --enable-extras=ssl_gnutls. after it is enabled i receve the error [*] Module file could not be found: m_ssl_gnutls.so after starting the server with ./run/inspircd start

**Steps to reproduce the issue:**

1.install InspIRCd version: 3.8.1 from source 
2.run ./configure --enable-extras=ssl_gnutls
3.run ./inspircd start

**Describe the results you received:**
I receive the error:
[*] Module file could not be found: m_ssl_gnutls.so

**Describe the results you expected:**
i expect the server to start a usual with module enabled

**Additional information you deem important (e.g. issue happens only occasionally):**
I looked into the run/modules folder because that is the location that I think m_ssl_gnutls.so should be located. it isn't
**Output of `./bin/inspircd --version`:**
InspIRCd version: 3.8.1

I searched the GitHub repo for this file but inly could find mentions of it, if anybody has access to this file it would be appreciated.

**Comment by sadiepowell** [comment]
You need to run --enable-extras before Make.  Try running `make install`.

**Comment by James-Seely-Mirrors** [comment]
I appreciate your swift response and am not receiving the following error when running make install: 
Error: unable to find the gnutls compiler flags for ssl_gnutls!
if anyone knows how to fix this it would be appreciated.

**Comment by sadiepowell** [comment]
You need to install the dependencies for the ssl_gnutls module.

i.e. the gnutls development packages and pkg-config.

**Comment by James-Seely-Mirrors** [comment]
thank you.
